main/src/ca/uwaterloo/flix/language/phase/Inliner.scala

```diff
@@ -17,12 +17,18 @@
 package ca.uwaterloo.flix.language.phase
```

Comment:
Add copyright header with your name (see pretty much any other file).
Ensure that you also add yourself to `AUTHORS.md`. :)